locales/en/messages.json

```diff
@@ -2519,6 +2519,41 @@
         "message": "<strong>I understand the risks</strong>, the propellers are removed - enable motor control and arming, and disable Runaway Takeoff Prevention."
     },
 
+    "motorOutputReorderDialogClose": {
+        "message": "Cancel"
+    },
+    "motorOutputReorderDialogAgree": {
+        "message": "Start"
+    },
+    "motorsRemapDialogTitle": {
+        "message": "Remap motors"
+    },
+    "motorOutputReorderDialogOpen": {
+        "message": "Remap motors"
+    },
+    "motorOutputReorderDialogSelectSpinningMotor": {
+        "message": "Click on the spining motor..."
```

Comment:
Can this be changed to be context aware, ie "Click on motor _n_..."?
This will help if a battery isn't used and user is mapping without spinning motors

Reply:
I don't know if it will be too long, but maybe something like `Testing motor {motor_number}. Click on the spinning motor...`

I'm not good at English but I think `spinning` is with double `n`?

Reply:
You are correct, spinning is double `n`.
I thought it might be best to leave out the word "spinning", as this can be performed without the motors actually spinning.
However, if you wish to keep it in, maybe: `Click on spinning motor {motor_number}...`

Reply:
@richardbulger

> You are correct, spinning is double `n`.
> I thought it might be best to leave out the word "spinning", as this can be performed without the motors actually spinning.
> However, if you wish to keep it in, maybe: `Click on spinning motor {motor_number}...`

@richard, @McGiverGim I think the motor number there can be a little bit misleading for a user.
Apparently he messed up the wiring (that's why mots of people will be using this "wizard"). so for him, "motor 3" could sound weird.

On a side note, check the picture. The motors user already clicked on are marked with green and there is no way to click on them again, they don't react also on mouse hover like the rest of the motors:
![image](https://user-images.githubusercontent.com/2925027/86162383-50d21600-bad4-11ea-8cd9-02df27d8dd13.png)

Reply:
I don't disagree with anything you are saying :) As a setup wizard that uses spinning motor to remap resources, it is perfect, and maybe wanting to add further functionality will add too much complexity.
Maybe further functionality to map known motors to could be expanded on down the line, in a different way

Reply:
Richard, thanks! Not trying to argue. If people think I should add a motor number - it is not a problem at all, you know :) Just wanted to give my biased perspective on that.

Reply:
No problem!
It would be interesting to get another perspective. Mine is also biased :)

Reply:
We can wait until merged to see the use that people is giving and feedback. Then we can decide if add a motor number.

Reply:
@McGiverGim let me know when I should apply changes you suggested here in the code review. Itching to have it merged lol

Reply:
When you have time. I will approve it but maybe it take some days to be merged. We have some PR pending and must be merged by mikeller.